assets/css/app.css

```diff
@@ -0,0 +1,68 @@
+body {
+  font-family: 'Poiret One', cursive;
+  font-size 1.6em
+}
+
+.form__button--submit {
+  width: 15%;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 0.5em;
+}
+
+.filters__button {
+  text-align: center
+}
+
+.filters__button--active   {
+  width: 15%;
+}
+
+.filters__button--all {
+  width: 15%;
+}
+
+.filters__button--complete {
+  width: 15%;
+}
+
+.total {
+  text-align: center;
+  font-size: 1.5em;
```

Comment:
This will be `1.5 * 1.6 * 16px`, which is tough. Try to avoid nested `em`s because the math is harrrrd